src/components/app-drawer/AppDrawer.tsx

```diff
@@ -19,12 +20,16 @@ const AppDrawer: FC<AppDrawerProps> = ({
   closeIcon = true,
   ...props
 }) => {
+  const { t } = useTranslation()
+
   const { checkConfirmation } = useConfirm()
 
   const handleCloseDrawer = async () => {
     const confirmed = checkConfirmation({
       message: 'questions.unsavedChanges',
-      title: 'titles.confirmTitle'
+      title: 'titles.confirmTitle',
+      confirmBtn: t('common.discard'),
+      cancelBtn: t('common.cancel')
```

Comment:
i am not sure that here we need to use t('') for buttons translations cause you already use it inside ConfirmDialog
<img width="545" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/90138904/5612dbdc-5f04-4601-81b7-2e7b46f706b6">

Reply:
ye, that's was at last commit, i forgot to take away from confirmdialog translation, right now its fixed